### PR TITLE
feat(space-classification): allow user to create a classified space

### PR DESCRIPTION
**Pull request by AMANDUA**
This PR comprises of code changes to allow user to create a classified space

Fixes https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-176840
---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js [hunk 1]
```diff
@@ -2306,6 +2306,14 @@ const Conversation = WebexPlugin.extend({
       }));
     }
 
+    if (!params.isDefaultClassification && params.classificationId) {
```
Comment (adamweeks) [inline packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js:2309]: Please add these additional params to the jsdoc on the params object on the `create` function near line 167.

**Review by adamweeks — CHANGES_REQUESTED** [review]
Thanks for adding this support. Could you please add an additional test on the `create` function that will call with this param?

**Comment by adamweeks** [comment]
@AMANDUA 

> I tried adding test with existing orgs but the integration tests are failing saying org is not having space classifications enabled...

If we aren't able to get any integration orgs for this feature, perhaps you can add a test that expects the error when the feature isn't enabled at least.

**Comment by AMANDUA** [comment]
> @AMANDUA
> 
> > I tried adding test with existing orgs but the integration tests are failing saying org is not having space classifications enabled...
> 
> If we aren't able to get any integration orgs for this feature, perhaps you can add a test that expects the error when the feature isn't enabled at least.

yes that sounds good to me...

**Comment by AMANDUA** [comment]
Added the test @adamweeks  
image

and passed the  CI.

But few integration tests are failing , saying timeout
image

**Review by adamweeks — APPROVED** [review]
Merging. The test failures are the same failures as master branch currently experiencing: https://app.circleci.com/pipelines/github/webex/webex-js-sdk/4124/workflows/7d5fae21-b726-48ca-ac1b-dc3714fb90c7/jobs/29488

**Pull request merged**